scripts/commands/playerCommands.js

```diff
@@ -11,24 +11,31 @@ const tp = (plr, p2) => {
   Call.setCameraPosition(plr.con, plr.unit().x, plr.unit().y);
 };
 
-const clean = (con) => {
-  Timer.schedule(
-    () => {
-      Call.sound(con, Sounds.rockBreak, 1, 1, 0);
-    },
-    0,
-    0.05,
-    10
-  );
-  Vars.world.tiles.eachTile((t) => {
-    if (
-      [
-        107, 105, 109, 106, 111, 108, 112, 117, 115, 116, 110, 125, 124, 103, 113, 114, 122,
-      ].includes(t.block().id)
-    ) {
-      t.setNet(Blocks.air, Team.sharded, 0);
-    }
-  });
+const Cleaner = {
+  clean(con) {
```

Comment:
Does this reference the clean function deleted above?

Reply:
It's the same function, just encapsulated in an object so the cooldown is tied to it.

Reply:
I need to stop reading so fast lol. I see it now. Yeah it looks good